src/components/Document/DocumentMatches.vue

```diff
@@ -10,25 +10,47 @@
 					Zhoda so súborom
 				</span>
 			</div>
-			<div
-				v-for="match in matches"
-				:key="match.name"
-				class="flex justify-between py-3 md:py-5 text-base md:text-lg text-black bg-white border-b-2 last:border-b-0 md:border-b-2 md:last:border-b-2 border-gray-400"
+			<a
+				v-for="(match, index) in matches"
+				:key="index"
+				@click="
+					$router.push({
+						name: `compare`,
+						params: {
+							result: resultId,
```

Comment:
Aren't we able to get the id's from the URL parameters instead of passing it through components using props?